tests/test_mpi.py

```diff
@@ -1055,5 +1152,7 @@ class TestIsotropicAcoustic(object):
     # TestSparseFunction().test_ownership(((1., 1.), (1., 3.), (3., 1.), (3., 3.)))
     # TestSparseFunction().test_local_indices([(0.5, 0.5), (1.5, 2.5), (1.5, 1.5), (2.5, 1.5)], [[0.], [1.], [2.], [3.]])  # noqa
     # TestSparseFunction().test_scatter_gather()
-    TestOperatorAdvanced().test_nontrivial_operator()
+    # TestOperatorAdvanced().test_nontrivial_operator()
```

Comment:
Debug remainder?

Reply:
yeah I'll comment it out

I'll leave the ``if __main__`` thing because with ``tmpi`` that's what I use for debugging